src/task2.py

```diff
@@ -0,0 +1,39 @@
+import base64
+from itertools import islice
+
+def getAverageLength(rdd, col):
+    # Remove header
+    rdd = rdd.mapPartitionsWithIndex(
+        lambda idx, it: islice(it, 1, None) if idx == 0 else it
+    )
+
+    # Find number of rows
+    numRows = rdd.count()
+
+    # Select correct column and decode base64
+    rdd = rdd.map(lambda line: len((base64.b64decode(line[col]))))
```

Comment:
Won't this still have the b'[text]' format? At least when I tried it I needed to use
`rdd = rdd.map(lambda line: len(base64.b64decode(line[col]).decode("utf-8")))`

Reply:
Tried to count it and it ignores the b''. At least I think so! But we can add the rest to be sure